### ISSUE TITLE
log_image support?

### ISSUE THREAD

**Issue by elgalu** [DAGsHub/client#125]
Similar to https://www.tensorflow.org/api_docs/python/tf/summary/image or https://mlflow.org/docs/latest/python_api/mlflow.tracking.html#mlflow.tracking.MlflowClient.log_image

**Comment by guysmoilov** [comment]
Thanks @elgalu , that is a suggestion we've been thinking about how to implement properly.
Why not just save the image as part of the commit, next to the output `metrics.csv` and `params.yml`?
You expect to see the image as part of the experiment dashboard in the end?

**Comment by elgalu** [comment]
> You expect to see the image as part of the experiment dashboard in the end?

Yes, even more, we expect to see each logged image in each step to track model evolution
